setup.py

```diff
@@ -18,7 +18,7 @@
     'pillow',
     'appdirs',
     'scooby>=0.5.1',
-    'vtk',
+    'vtk>=8.1.2,<9.3',
```

Comment:
Does `setup.py` support `<=9.2.0.*`?

Reply:
I was having trouble permitting a release candidates with any version range unless you specifically permit it, as in:
https://github.com/pyvista/pyvista/blob/88a894084bfb93ddad4a8e38f9856f1b182a3c04/setup.py#L21-L22

What I did was explicitly permit the release candidate wheel, but only for Python 3.10. Personally, I don't feel that release candidates should be permitted by default, but only in special circumstances like general unavailability otherwise as in the case of Python 3.10

Reply:
This makes sense to me.  Ideally the rc part could be removed before the release in pyvista.  If it can't, should we use some form of `==9.2.0.*`, so that later release candidates can be used?  I'm not sure.  Maybe best to pin it and manually keep updating it until the release?  This way there are no surprises.

Reply:
> If it can't, should we use some form of ==9.2.0.*, so that later release candidates can be used?

If someone knows how to get release candidates working with a version range, please let me know. Otherwise, I'll permit `==9.2.0rc` for PyVista for now.

Reply:
See my other suggestion, if that doesn't work, I'm in favor of `==9.2.0rc1` and manually bumping it as needed, and hopefully removing it before next release (vtk had a 1 month release cycle last time).

Reply:
Agreed then. Let's keep to this for now and hopefully 9.2.0 is out before the next release of PyVista.